xsuite/src/world/lsworld.test.ts

```diff
@@ -235,7 +239,9 @@ test.concurrent("LSWorld.getAccount", async () => {
 
 test.concurrent("LSWorld.getAllSerializableAccounts", async () => {
   using world = await LSWorld.start();
-  const { wallet, wallet2, wallet3, contract } = await createAccounts(world);
+  const { wallet, wallet2, wallet3, contract } = await createAccounts(world, {
```

Comment:
In this test, let's not use `createAccounts` and just create manually a wallet and a contract owned by the wallet. And contract has the dummy code.